dao.py

```diff
@@ -540,6 +547,37 @@ def remove_region(self, region):
         if self.regions_col.find_one({'display_name': region.display_name}):
             self.regions_col.remove(region.get_json_dict())
 
+    def update_region_ranking_criteria(self, region_id,
+                                       ranking_num_tourneys_attended,
+                                       ranking_activity_day_limit):
+                                       #tournament_qualified_day_limit):
+        if self.regions_col.find_one({'_id': region_id}):
+            self.regions_col.update({'_id': region_id},
+                                    {'$set':
+                                        {
+                                         'ranking_num_tourneys_attended': int(ranking_num_tourneys_attended),
+                                         'ranking_activity_day_limit': int(ranking_activity_day_limit)
+                                         #'tournament_qualified_day_limit': int(tournament_qualified_day_limit)
+                                        }
+                                     })
+
+
+    def get_region_ranking_criteria(self, region_id):
+        print region_id
+        if self.regions_col.find_one({'_id': region_id}):
+            result = M.Region.load(self.regions_col.find_one({'_id': region_id}))
+
+            print result
+
+            print self.regions_col.find_one({'_id': region_id})
+
+            region_ranking_criteria = {}
```

Comment:
basically, instead of manually serializing region_ranking_criteria (lines 574-579) you can just directly `return result.dump(context='web')`. (this should accomplish exactly what you have but also return the region display_name; if you want to exclude that for some reason, i think you can do `result.dump(context='web', exclude=['display_name'])`).